app/test_main.py

```diff
@@ -15,6 +15,17 @@
         (27, 27, [2, 2]),
         (28, 28, [3, 2]),
         (100, 100, [21, 17])
+    ],
+    ids=[
+        "should return correct result",
+        "should return correct result",
+        "should return correct result",
+        "should return correct result",
+        "should return correct result",
+        "should return correct result",
+        "should return correct result",
+        "should return correct result",
+        "should return correct result"
     ]
 )
 def test_correct_result(cat: int, dog: int, result: list) -> None:
```

Comment:
and here you can try to rename func to something like: `test_cat_dog_age_should_equal_humans`